coralproject/talk
# [CORL-2257]: Reject all comments by user for single-site moderators

## What does this PR do?

These changes make it so that existing comments can be rejected across single/specific sites, and not just across all sites for a user. So, the copy for rejecting existing comments has been updated for the all/specific sites options for admin and org moderators. And for single-site moderators, the option to reject existing comments across their single site has been added. These changes are for https://vmproduct.atlassian.net/browse/CORL-2257.

## What changes to the GraphQL/Database Schema does this PR introduce?

None.

## How do I test this PR?

 You can test this PR by going to the `Community` tab in the admin, and then under the Status dropdown, selecting `Manage Ban`. Do this as an admin, org moderator, and site moderator (multi- and single-site). See that the reject all existing comments option is visible and unchecked by default in all cases. The copy should be different for the all sites, specific sites, and single-site options. See that when you check the reject all existing comments option in different cases, it rejects comments across the selected sites only. See that the reject all existing comments for all sites still works as expected, too.
 
## How do we deploy this PR?
